documentation/guides/migrating-from-v4-to-v5.md

```diff
@@ -0,0 +1,224 @@
+# Migrating from v4 to v5
+
+Polaris v5.0.0 ([full release notes](https://github.com/Shopify/polaris-react/releases/tag/v5.0.0)) removes most features deprecated in v4.x.x releases. This file describes all code updates required to stay up to date.
+
+## AppBridge integrations removed
+
+In v3 and v4 `AppProvider` could accept `apiKey` `shopOrigin` and `forceRedirect` props that would configure AppBridge delegation in our `ResourcePicker`, `Loading`, `Modal`, `Page` and `Toast` components. This functionality has been removed.
+
+In v5 AppBridge integration has been removed from these components:
+
+- The `AppProvider` component's `apiKey`, `shopOrigin` and `forceRedirect` have been removed.
```

Comment:
```suggestion
- The `AppProvider` component’s `apiKey`, `shopOrigin` and `forceRedirect` have been removed.
```
I'm not sure this is important in markdown files but I try to use curly quotes for public facing docs